### ISSUE TITLE
ROW General Fee Waiver Report (TL)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#3168]
##ROW General Fee Waiver Report
- Copy #3167 and change the report parameters. 
    -  User able to query by adjustment Start Date, Adjustment End Date

-------

### ROW General Fee Waiver Custom Report

**Stakeholders:** Paloma, Kim
**DTS Manager:** Tracy

- Menu Report
- Make it exportable for data analysis
- Report Parameters:   Adjustment Start Date, Adjustment End Date 
- Folder Types to include in query: EX, RW (exclude street event subtype), DS, LC, ECV (transportation   subtype), PSM, GF (transportation subtype)

**Report Data:**
- Folder RSN
- Folder Type
- Adjustment Reason
- Amount Adjusted
- Adjustment Comment
![image](https://user-images.githubusercontent.com/49930868/85894245-fe35e880-b7b9-11ea-86f1-ba6d8d511479.png)

**Comment by TracyLinder** [comment]
Development will begin sprint 29 or 30.

**Comment by TracyLinder** [comment]
It has been determined that one menu report can meet requirements for both this request and #3167 . We will continue tracking the work on 3167, and close this issue.

**Comment by Sreelatha-BlazeKannan** [comment]
@TracyLinder, @Nadin-Nader - Subha will be working on this.

**Comment by TracyLinder** [comment]
LaunchIT will begin working on this after #3167 is approved by stakeholders.

**Comment by shroffjs** [comment]
Status: Sent the demo spreadsheet to @TracyLinder @Nadin-Nader @Sreelatha-BlazeKannan, in email for review on 10/28/2020.
=============================================================
Hi Tracy/Nadin,

I worked on #3168 to replicate as #3167, but with adjustment date(Start Date , End Date) as parameters. Here is the sample output attached for your review. The data(Adjustment Date between: 10/1/2018 – 10/26/2020) in the spreadsheet is generated from Crystal Reports Tool connected to TEST database. It’s not yet updated on AMANDA. Please let me know your feedback so that we can move to DEV/TEST.

Thanks,
Subha

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Waiting on feedback to move  to DEV / TEST

**Comment by TracyLinder** [comment]
@SubhaShroff the format looks good, please promote.
### Approved to promote to DEV/Test 🎉

![image](https://user-images.githubusercontent.com/49930868/98175533-690bd480-1ebc-11eb-8cfc-2cf80b36bb74.png)

**Comment by shroffjs** [comment]
Thank you @TracyLinder.
@Nadin-Nader I have sent you the Deployment Plan and .rpt file today(11/5/2020).

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @SubhaShroff @TracyLinder I have deployed this to DEV and TEST. Please UAT and let me know if all is well.

**Comment by shroffjs** [comment]
Thank you @Nadin-Nader 
Tested. 'FOLDERTYPE' column is also displayed, thats not part of AC. Updated the report not to display it and sent you in email. Please update in servers.

**Comment by shroffjs** [comment]
@TracyLinder @Nadin-Nader @Sreelatha-BlazeKannan 
Tested after update on DEV and TEST. Report: ROW General Fee Waiver Custom Report(99011) is working as expected. Its ready for  UAT.

**Comment by TracyLinder** [comment]
Per Paloma, I can promote to prod without UAT from stakeholders since this is a duplicate of another report already working well.

**Comment by shroffjs** [comment]
@TracyLinder @Nadin-Nader @Sreelatha-BlazeKannan I sent the PROD Deployment Plan and .rpt file to Nadin.

**Comment by Sreelatha-BlazeKannan** [comment]
@TracyLinder @Nadin-Nader - If its moved to PROD and it  can be closed.

**Comment by Nadin-Nader** [comment]
@SubhaShroff @Sreelatha-BlazeKannan @TracyLinder  I have sent this over to the CTM Team for Deployment to PROD under TASK **SCTASK0104269**

**Comment by TracyLinder** [comment]
This was moved to prod on 11/10. I will update stakeholders & close. Great job team!
